music-app.tsx

```diff
@@ -10,16 +10,19 @@ import {
   playTrack as playSpotifyTrack,
 } from "./spotify.ts";
 
+import "https://deno.land/std@0.213.0/dotenv/load.ts";
```

Comment:
We already load once in `spotify.ts`. This can be removed.

Reply:
Oh nice catch